content/docs/reference/glossary.md

```diff
@@ -36,13 +36,17 @@ A periodic load generated by the Control Plane to determine if a compute can sta
 
 A [copy-on-write](#copy-on-write) clone of a Neon project's primary branch or previously created child branch. A branch can be created from the current or past state of the parent branch. A branch created from the current state of the parent branch includes the databases and users that existed in the parent branch at the time of branch creation. A branch created from a past state of the parent branch includes the databases and users that existed in the past state. The data in a branch can be modified independently from its originating data. See [Branching](/docs/introduction/branching). Each branch has a dedicated [compute endpoint](#compute-endpoint), which is the compute instance associated with the branch. Connecting to a database in a branch requires connecting via the branch's compute endpoint. For more information, see [Connect to a branch](/docs/manage/branches#connect-to-a-branch).
 
+### Child branch
+
+A [branch](#branch) created from a [primary branch](#primary-branch) or another branch in your Neon project.
```

Comment:
a child branch can have it's own child
this definition doesn't work well for the children of direct descendants of primary branch

Reply:
Changed definition to "non-primary branch"